index.bs

```diff
@@ -350,7 +350,7 @@ Report long tasks {#report-long-tasks}
             1. Set |newEntry|'s {{PerformanceEntry/name}} attribute to |name|.
             1. Set |newEntry|'s {{PerformanceEntry/entryType}} attribute to "<code>longtask</code>".
             1. Set |newEntry|'s {{PerformanceEntry/startTime}} attribute to |start time|.
-            1. Set |newEntry|'s {{PerformanceEntry/duration}} attribute to |end time| minus |start time|.
+            1. Set |newEntry|'s {{PerformanceEntry/duration}} attribute to the integer part of |end time| minus |start time|.
```

Comment:
This is ambiguous - I think you mean to round (|end time| - |start time|), but it's not clear from your phrasing. 
Can you clarify it?

Reply:
It's kind of getting old to keep pointing this out but you can't subtract the precise time then round the result to 1ms precision. You need to reduce the precision of each time first and then subtract. Otherwise, you reveal sub-millisecond timing at arbitrary precision by way of adjusting the starting time.

Reply:
Apologies, but I don't recall you already pointing that out to me. At the same time, I'd appreciate a more amicable & respectful tone in the future (towards myself and others).

As the start and end times input into the processing model by [HTML](https://html.spec.whatwg.org/#event-loop-processing-model) are not arbitrary times (they are times we get from HR-time, which are already clamped, although we're in the process of clarifying HR-time further), I don't think that doing the rounding before the subtraction would expose any more arbitrary position than `performance.now()` would.

You're right that where we perform the rounding is meaningful: rounding before subtraction would result in more precise time in cases where the fractional part of start time is larger than the fractional part of end time.
I'm not sure it matters much either way.

Reply:
I agree with Yoav. Given that a) we don't round the start time exposed and b) you cannot control the start time in a precise way, it seems to me that this is fine.